src/pages/settings/Wallet/WalletPage/WalletPage.tsx

```diff
@@ -538,10 +536,14 @@ function WalletPage({bankAccountList = {}, cardList = {}, fundList = {}, isLoadi
                                 {isPopoverBottomMount && (
                                     <MenuItem
                                         title={paymentMethod.formattedSelectedPaymentMethod.title}
-                                        icon={paymentMethod.formattedSelectedPaymentMethod.icon}
+                                        icon={paymentMethod.formattedSelectedPaymentMethod.icon?.icon}
+                                        iconHeight={paymentMethod.formattedSelectedPaymentMethod.icon?.iconHeight}
```

Comment:
@dukenv0307 Why are we passing iconHeight and iconWidth separately? Its already part of `paymentMethod.formattedSelectedPaymentMethod.icon` right? Why are we splitting the params and passing?

Reply:
Sorry, nevermind. Its under conditional rendering. Wouldn't be helpful